ArraySorting.cls

```diff
@@ -0,0 +1,96 @@
+public class ArraySorting{
+	public static void ascendingOrder(List<Integer> values){
+		List<Integer> integerArray = new List<Integer>(values);

```

Comment:
What is need to add `values` to another list. You can simply use `values` as it is list itself.

- [ ] Use values remove unnecessary variable declaration